pphysx3d/src/scene/mod.rs

```diff
@@ -110,7 +155,7 @@ impl PhysicsScene {
         for object in &mut self.objects {
             // If object is immovable, aka object has infinite mass, then don't apply gravity (as it would be an infinite force)
             if object.inv_mass() > f32::EPSILON {
-                object.add_force(gravity * object.mass());
+                object.add_force(time_step * gravity * object.mass());
```

Comment:
This is so that gravity is reasonably scaled. Now when I think about it I think this is the wrong place to scale it in but it makes for a way more realistic simulation with my current object parameters.